src/backend/controllers/tankController.js

```diff
@@ -107,21 +97,15 @@ exports.casusUpdate = async (req: Request, res: Response) => {
 			.json({ errors: errors.array() });
 	}
 
-	await Tank.findById(req.params.tankId, function (err, tank) {
-		if (err){
+	await Tank.findById(req.params.tankId, function (err, tank){ 
+		if(err){
```

Comment:
Not sure why there were changes here. And even so, as picky as it is, spacing between things are off.

Reply:
idk, not really optional in my book. It isn't hard to look through your own stuff before a PR, so make it match the style guide please.